### DISCUSSION TITLE
Making Sense of the Sensors

### DISCUSSION THREAD

**Discussion by modishnouns** [Rem0o/FanControl.Releases#2686] [Q&A] [answered]
Hi all,

How do I determine which temperature source is the main CPU temp?
In the attached screenshot you can see the NZXT CAM overlay (top left) and below that HWINFO. Both are reading 84 deg. CPU temp.
To the right, I have Fan Control with a temperature source drop-down displayed and none of the sources look like my current CPU temp.
Any ideas what's going on?

![CPUTemps](https://github.com/user-attachments/assets/6b24e5ca-5f62-4431-b63f-076f55181449)

**Comment by Rem0o** [comment, marked as answer]
There is no CPU sensor in there.   What CPU do you got and do you have "CPU" enabled in sensor sources in the settings page?

**Reply by modishnouns** [reply]
Aha! I just needed to enable it as a source. Thank you!
